.gitignore

```diff
@@ -17,3 +17,5 @@ vault.txt
 *.swp
 *.svg
 server/www/img/icons/
+vendor/*
```

Comment:
Could we be explicit about this being a customizations dir? ie call it `custom/` 
It doesn't serve the same role as `vendor` since thats already taken by `node_modules` but usually (ie. in PHP) vendor = installed packages.